Aptoide/aptoide-client-v8
# [bugfix/ASV-1696] Click on Apps cards

**What does this PR do?**

   This PR fixes app cards clicks on Apps. Before, only updates not in progress actually did anything. Now clicking any state of an update card / active download card goes to the app view of the app.

Note that it was decided that analytics wouldn't change.

**Database changed?**

   No


**How should this be manually tested?**

  Test the click of every state of Update/Active Download card. In any state, it should go the app view. Installed card does not apply.

**What are the relevant tickets?**

  [ASV-1696](https://aptoide.atlassian.net/browse/ASV-1696)


**Code Review Checklist**
